client/src/main/java/io/pravega/client/stream/impl/Pinger.java

```diff
@@ -66,11 +75,11 @@ void stopPing(UUID txnID) {
 
     private long getPingInterval(long txnLeaseMillis) {
         double pingInterval = txnLeaseMillis * PING_INTERVAL_FACTOR;
-        if (pingInterval < TimeUnit.SECONDS.toMillis(5)) {
+        if (pingInterval < TimeUnit.SECONDS.toMillis(10)) {
```

Comment:
I have seen this number (`10`) used in several places. Please, consider using a constant to refer to it.

Reply:
done.